packages/react/package.json

```diff
@@ -33,5 +33,8 @@
     "rollup-plugin-visualizer": "^5.12.0",
     "typescript": "^5.2.2",
     "vite": "^5.2.0"
+  },
+  "bosConfig": {
```

Comment:
Shall we call this `deployConfig` to avoid `bos` acronym?

Reply:
I'd prefer that it's clearly a Near-related field to make it clear it's user-defined and avoid any potential collisions.

Reply:
yeah let's please avoid any `bos` references 😅 

`nearDeployConfig`?